docs/assets/css/style.scss

```diff
@@ -3,17 +3,27 @@
 
 @import "{{ site.theme }}";
 
+.topnav input[type=text]  {
+  width: 100%;
+  font-size: 18px;
+  padding: 11px;
+  border: 1px solid #ddd;
+}
+
+
 .navigation h1 {
   font-size:14px;
   float:none;
-	margin-bottom: 12px;
+	margin-bottom: 6px;
 }
 
 .navigation li {
-  width:100%;
+
```

Comment:
Cut the whitespaces.  And in the future please do not add so much unnecessary whitespace. It just shows up in the diffs as a change  that a code reviewer then wastes time on ;)